src/modules/dashboard/components/UnclaimedTransfers/UnclaimedTransfersItem.tsx

```diff
@@ -1,30 +1,18 @@
 import React, { useCallback } from 'react';
-import {
-  FormattedMessage,
-  defineMessages,
-  FormattedDateParts,
-} from 'react-intl';
+import { FormattedMessage, defineMessages } from 'react-intl';
 
 import { ActionButton } from '~core/Button';
-import MaskedAddress from '~core/MaskedAddress';
 import Numeral from '~core/Numeral';
-import Icon from '~core/Icon';
-import EthUsd from '~core/EthUsd';
-
-import {
-  useTokenQuery,
-  ColonyTransaction,
-  useUsernameQuery,
-  useLoggedInUser,
-} from '~data/index';
+import TokenIcon from '~dashboard/HookedTokenIcon';
+
+import { useTokenQuery, ColonyTransaction, useLoggedInUser } from '~data/index';
 import { ActionTypes } from '~redux/index';
 import { mergePayload } from '~utils/actions';
 import { getTokenDecimalsWithFallback } from '~utils/tokens';
 import { checkIfNetworkIsAllowed } from '~utils/networks';
 
-import { tokenIsETH } from '../../../core/checks';
-
 import styles from './UnclaimedTransfersItem.css';
+import CopyableAddress from '~core/CopyableAddress';
```

Comment:
shouldnt this import be located in the block above?

Reply:
You are correct as always Mr. Dan